streamer/src/quic.rs

```diff
@@ -51,15 +51,15 @@ pub const DEFAULT_QUIC_ENDPOINTS: usize = 1;
 pub const DEFAULT_TPU_COALESCE: Duration = Duration::from_millis(5);
 
 pub fn default_num_tpu_transaction_forward_receive_threads() -> usize {
-    num_cpus::get()
+    num_cpus::get().min(16)
```

Comment:
not sure why are you so generous to forwarding. I would say it needs less than normal tpu traffic.

Reply:
not really, fwd is where most high value txs come from